code/game/objects/objs.dm

```diff
@@ -15,7 +15,8 @@ var/global/list/reagents_to_log = list("fuel"  =  "welder fuel", "plasma"=  "pla
 
 	var/damtype = "brute"
 	var/force = 0
-
+	var/wrenchable = 0 //can the object be secured using a wrench?
```

Comment:
I'd rather this was a proc than a var, because setting wrenchable on machines is going to do absolutely nothing and it'll confuse people.

You can do `wrenchable()` or `can_wrench()` and override it to return 1 for what you want.